### DISCUSSION TITLE
passive login failed

### DISCUSSION THREAD

**Discussion by derekgdicamillo** [OctoFarm/OctoFarm#1354] [Q&A]
I am trying to connect octofarm to octoprint. I cant seem to get connected. I keep getting an error stating passive login failed. when I connect it is able to pull the printer name and I am able to even move the print head. But can't get it to fully connect. I have disabled all plugins and logged out on the webclient. Still fails to connect. I have tried an application api key and user api key.

**Comment by NotExpectedYet** [comment]
I will need logs to help with this please

**Reply by derekgdicamillo** [reply]
I thought that maybe it was because I was installing on a vm but i just completed install on windows laptop and I am getting the same errors

**Comment by derekgdicamillo** [comment]
here are my logs
[OctoFarm.Access (1).log](https://github.com/OctoFarm/OctoFarm/files/10264059/OctoFarm.Access.1.log)
[OctoFarm.Printer.OctoPrint.log](https://github.com/OctoFarm/OctoFarm/files/10264060/OctoFarm.Printer.OctoPrint.log)
[pm2.out.log](https://github.com/OctoFarm/OctoFarm/files/10264061/pm2.out.log)
[OctoFarm.System.log](https://github.com/OctoFarm/OctoFarm/files/10264062/OctoFarm.System.log)

**Reply by NotExpectedYet** [reply]
228 is timing out, so your OP instance isn't responding within 10 seconds. You've either a really really slow network or something else with the systems/network. I'd start by assuring that the octofarm host can actually contact octoprints host.

It shouldn't make a difference VM or not but do you have any weird routing / VLAN setups going on? Have you tried to connect another device to OctoPrint? 

Infact I've just found a bug: HTTP MESSAGE: Promise timeout threw an error! : {"error":"Error: Timed out in 5000ms.","url":"http://192.168.1.228/","method":"GET"}  That should be 10 seconds too but shouldn't be related to your issue as a restart usually resolves the bug.

**Comment by derekgdicamillo** [comment]
I don't have any weird routing setups.

I am able to connect via octoanywhere and octoapp. I did try booting
octoprint into safemode in case it was a plug-in causing an issue.

Just can't figure out what could be causing it. I do have decent wifi. I
can stream the video feed from other apps.

**Reply by NotExpectedYet** [reply]
I'm confused myself, there's nothing else in the logs apart from the timeout issue so I'm unsure what to suggest.

**Reply by NotExpectedYet** [reply]
If your pulling the printer name then the API is responding so I'd gather it could possibly be something OctoPrints end. 

Are there anything in it's logs?

**Comment by derekgdicamillo** [comment]
I understand I'll keep working at it. Thank you for trying!

**Reply by NotExpectedYet** [reply]
Did you see my message about OctoPrint logs? 

It looks like some of the api calls do go through. It only seems to error out when it's setting up the websocket + the additional suplemental calls like file info. 

Do you have multiple users on OctoPrint?

**Comment by derekgdicamillo** [comment]
I didn't see the message. I'll pull those and send them. I have 3 printers
on three seperate pi's.

**Reply by NotExpectedYet** [reply]
I gather all 3 Pi's do the same?

**Comment by derekgdicamillo** [comment]
Yes they do.

**Reply by NotExpectedYet** [reply]
Do you have additional logs to send please? 

There was barely any connection attempts in the others.
